src/levanter/compat/hf_checkpoints.py

```diff
@@ -643,6 +668,23 @@ def _save_pretrained_local(
             else:
                 raise
 
+        if self.tokenizer:
+            dict_config = mergedeep.merge(
+                {},
+                dict_config,
+                {
+                    "pad_token_id": self.tokenizer.pad_token_id,
```

Comment:
not all tokenizers have a pad_token_id (or eos or bos), can you gate these behind whether or not they're there? or maybe -1 will be ok i dunno

Reply:
Gated!

Reply:
Confirmed the None checks won't throw errors, HuggingFace defines all these properties in the Base Tokenizer: https://github.com/huggingface/transformers/blob/v4.38.2/src/transformers/tokenization_utils_base.py#L1165-L1183

Reply:
i meant on the loading side

Reply:
lgtm!